src/main/java/mate/academy/internetshop/lib/Injector.java

```diff
@@ -7,16 +7,19 @@
 import java.util.ArrayList;
 import java.util.Enumeration;
 import java.util.List;
+import org.apache.log4j.Logger;
 
 public class Injector {
+
     private static final String PROJECT_MAIN_PACKAGE = "mate.academy.internetshop";
+    final static Logger logger = Logger.getLogger(Injector.class);
     private static List<Class> classes = new ArrayList<>();
 
     static {
         try {
             classes.addAll(getClasses(PROJECT_MAIN_PACKAGE));
         } catch (ClassNotFoundException | IOException e) {
-            e.printStackTrace();
+            logger.error("System failure!", e);;
```

Comment:
@RUSLAN-SIMAKOV Якщо ти не заінжектиш депенденсі - твоя програма не запустить корректно - тому тут відловлювати і ігнорувати помилки - погана практика. Краще викинути RuntimeException
```suggestion
            throw new RuntimeException("Can't initialize dependencies", e);
```